casepro/cases/models.py

```diff
@@ -458,7 +458,8 @@ def as_json(self, full=True):
                 'labels': [l.as_json(full=False) for l in self.labels.all()],
                 'summary': self.summary,
                 'opened_on': self.opened_on,
-                'is_closed': self.is_closed
+                'is_closed': self.is_closed,
+                'msg_backend_id': self.initial_message.backend_id if self.initial_message else None,
```

Comment:
This should be called `initial_message` since cases have many messages and also the fact that we use backend ids is a hangover from the past that we should fix so I wouldn't put that in the name.

It should also be prefetched to avoid repeat db hits when listing cases.

However see subsequent comments about whether this field is needed.